### PR TITLE
Changed encoding from :raw to :LINEAR16

### PR DESCRIPTION
**Pull request by sgreenberg**
See https://github.com/GoogleCloudPlatform/ruby-docs-samples/issues/166

### CODE REVIEW

**Comment by remi** [comment]
Please see list of acceptable values for encoding parameter:
https://googlecloudplatform.github.io/google-cloud-ruby/#/docs/google-cloud/v0.28.0/google/cloud/speech/audio?method=encoding-instance

The encoding in question here:
 - `raw` - Uncompressed 16-bit signed little-endian samples. (LINEAR16)

Other examples:
 - `mulaw` - 8-bit samples that compand 14-bit audio samples using G.711 PCMU/mu-law. (MULAW)
 - `amr` - Adaptive Multi-Rate Narrowband codec. (sample_rate must be 8000 Hz.) (AMR)
 - `amr_wb` - Adaptive Multi-Rate Wideband codec. (sample_rate must be 16000 Hz.) (AMR_WB)
 - ...

If there is a question about semantics and what encoding values should be, this feedback should be filed [here](https://github.com/GoogleCloudPlatform/google-cloud-ruby/issues) ([google-cloud-ruby](https://github.com/GoogleCloudPlatform/google-cloud-ruby/issues))

**Comment by remi** [comment]
Note: these keys map to these underlying enum flags in the API specification:
https://github.com/googleapis/googleapis/blob/master/google/cloud/speech/v1/cloud_speech.proto#L138

**Comment by remi** [comment]
Blocking this issue on google-cloud-ruby issue [#1426](https://github.com/GoogleCloudPlatform/google-cloud-ruby/issues/1426) to change the name of the encoding key.

Note: this will be `:linear_16` not `:LINEAR16`

**Comment by remi** [comment]
Multiple aliases are [already available](https://github.com/GoogleCloudPlatform/google-cloud-ruby/blob/52b512b09ac5070a6c038f058793ecaa1f1267fe/google-cloud-speech/lib/google/cloud/speech/project.rb#L625-L630)), unblocking.

File: speech/quickstart.rb [hunk 1]
```diff
@@ -26,7 +26,7 @@
 file_name = "./audio_files/audio.raw"
 
 # The audio file's encoding and sample rate
-audio = speech.audio file_name, encoding:    :raw,
+audio = speech.audio file_name, encoding:    :LINEAR16,
```
Comment (remi) [inline speech/quickstart.rb]: Please update to `:linear16`

Comment (frankyn) [inline speech/quickstart.rb]: I'm making the change  and will commit once my local tests pass.

**Pull request merged**